src/pages/Farm/FarmOverview.tsx

```diff
@@ -61,13 +59,18 @@ FarmOverviewProps): JSX.Element {
               <TokenIcon symbol="WETH" alt="weth" />
             </>
           ) : (
-            farmTokens.map((token) => (
-              <TokenIcon
-                key={token.symbol}
-                symbol={token.symbol}
-                alt={token.symbol}
-              />
-            ))
+            poolTokens?.map((tokenAddress) => {
+              if (!chainId) return <div></div>
```

Comment:
Possible move and handle this `!chainId` check higher up? Outside of the loop?

Reply:
good call out!